components/dot_menu/dot_menu.tsx

```diff
@@ -224,11 +250,11 @@ export default class DotMenu extends React.PureComponent {
         </Tooltip>
     )
 
-    refCallback = (menuRef) => {
+    refCallback = (menuRef?: any) => { // TODO use more specific type
```

Comment:
I think the type of `menuRef` is
```suggestion
    refCallback = (menuRef?: Menu) => {
```
since we're accessing the `rect` method of it